cwms-servers/CWMS_HOME/bin/loggingSetup.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright (c) 2024
```

Comment:
Why did you separate this from logWatcher? What is the re-use?

Reply:
@adamkorynta

Reply:
loggingSetup.py can be used by any python script, such as instapost